clients/privacy-center/app/server-environment.ts

```diff
@@ -0,0 +1,206 @@
+        : config.server_url_production || (config as any).fidesops_host_production;
+
+    _environment.fidesApiUrl = legacyApiUrl;
+  }
+
+
+  return _environment;
+}
+
+/**
+ * Load the config.json file from the given URL, or fallback to default filesystem paths.
+ * 
+ * Loading precedence is:
+ * 1) Load from `configJsonUrl` argument
+ * 2) Load from file:///app/config/config.json on filesystem (absolute path)
+ * 3) Load from file:./config/config.json on filesystem (relative path)
+ * 
+ * NOTE: The "/app/config" path is for backwards-compatibility with the initial
+ * version of ethyca/fides-privacy-center, which expected to always load the
+ * configuration file from this well-known path.
+ */
+export const loadConfigFromFile: (configJsonUrl?: string) => Promise<Config | undefined> = async (configJsonUrl?: string) => {
+  const urls = [
+    configJsonUrl,
+    "file:///app/config/config.json",
+    "file:./config/config.json",
+  ];
+  const file = await loadConfigFile(urls);
+  if (file) {
+    const config = JSON.parse(file) as Config;
+    // TODO: validate config?
+    return config;
+  }
+}
+
+/**
+ * Load the config.css file from the given URL, or fallback to default filesystem paths.
+ * 
+ * Loading precedence is:
+ * 1) Load from `configCssUrl` argument
+ * 2) Load from file:///app/config/config.css on filesystem (absolute path)
+ * 3) Load from file:./config/config.css on filesystem (relative path)
+ * 
+ * NOTE: The "/app/config" path is for backwards-compatibility with the initial
+ * version of ethyca/fides-privacy-center, which expected to always load the
+ * configuration file from this well-known path.
+ */
+export const loadStylesFromFile: (configCssUrl?: string) => Promise<string | undefined> = async (configCssUrl?: string) => {
+  const urls = [
+    configCssUrl,
+    "file:///app/config/config.css",
+    "file:./config/config.css",
+  ];
+  const file = await loadConfigFile(urls);
+  return file;
+}
+
+/**
+ * Load a config file from the given list of URLs, trying them in order until one is successfully read.
+ */ 
+const loadConfigFile: (urls: (string | undefined)[]) => Promise<string | undefined> = async (urls: (string | undefined)[]) => {
+  // Dynamically import the "fs" module to read from the filesystem. This module
+  // doesn't exist in the browser context, so to allow the bundler to function
+  // we provide a (non-functional) fallback for "fs" in the webpack config (see
+  // next.config.js)
+  const fsPromises = (await import("fs")).promises;
+  if (!fsPromises) {
+    throw new Error("Unable to load 'fs' module!");
+  }
+
+  for (const urlString of urls) {
+    try {
+      if (!urlString) {
+        continue;
+      }
+      const url: URL = new URL(urlString);
+      // DEFER: add support for https:// to fetch remote config files!
+      if (url.protocol != "file:") {
+        throw new Error(`Config file URLs currently must use the 'file:' protocol: ${urlString}`);
+      }
```

Comment:
So this is worth mentioning here: as a follow-up, we could update the Privacy Center to support fetching the config files from a remote location instead of just the file system. This includes:
* downloading from an S3 bucket
* fetching from the Fides API

The latter option has always been the direction I wanted us to go for this.